lib/Elastica/Processor/DateIndexName.php

```diff
@@ -53,7 +53,7 @@ public function setDateRounding(string $dateRounding)
      *
      * @return $this
      */
-    public function setDateFormats(array $formats)
+    public function setDateFormats(array $formats): self
```

Comment:
default value should be --> yyyy-MM-dd’T'HH:mm:ss.SSSZ

Reply:
Can it be directly `ISO8601`?

Reply:
yep 👍